addon/types/model_export/model.py

```diff
@@ -172,14 +177,34 @@ def generate_qc(self):
             qc.write('$mostlyopaque')
             qc.write('\n')
 
+        if self.transform_source == 'MANUAL':
+            origin_x = self.origin_x
+            origin_y = self.origin_y
+            origin_z = self.origin_z
+            rotation = self.rotation
+            scale = self.scale
+        elif self.transform_source == 'OBJECT' and self.transform_object is not None:
+            loc, rot, sca = self.transform_object.matrix_world.decompose()
+            origin_x = -loc.x
+            origin_y = -loc.y
+            origin_z = -loc.z
+            rotation = degrees(rot.to_euler().z)
```

Comment:
If we're negating the location, should we negate rotation too?

Reply:
I don't remember any issues with rotation that required that, besides the fact it rotates the whole thing *after* the transformation, so it throws everything off. 😠